modules/login/php/passwordreset.class.inc

```diff
@@ -72,8 +72,7 @@ class PasswordReset extends \NDB_Form
                 // reset the password in the database
                 // expire password so user must change it upon login
                 $user->updatePassword(
-                    new \Password($password),
-                    new \DateTime('1999-01-01')
+                    new \Password($password)
```

Comment:
Hi @johnsaigle the updatePassword function in php/libraries/User.class.inc has the $expiry being option but it seems to get set 6 months from now if not supplied. Is this correct with what you're trying to do with this PR?

Reply:
This PR also contains changes to the User class where $expiry is removed in favour of $expired. This new variable represents a bool instead of a date.